libraries/cms/application/helper.php

```
@@ -231,7 +231,7 @@ public static function parseXMLLangMetaFile($path)
 		JLog::add('JApplicationHelper::parseXMLLangMetaFile is deprecated. Use JInstaller::parseXMLInstallFile instead.', JLog::WARNING, 'deprecated');
 
 		// Read the file to see if it's a valid component XML file
-		$xml = simplexml_load_file($path);
+		$xml = @simplexml_load_file($path);
```

Codestyle rules generally discourage the use of `@` to suppress errors.  Personally, I'd suggest a `file_exists($path)` check before running the `simplexml_load_file()` function.  I'd also patch `JInstaller::parseXMLInstallFile()` with the same change.